pythia6.sh

```diff
@@ -1,59 +1,77 @@
+# a pythia6 recipe based on the one from FairROOT
```

Comment:
Nice cleanup  :). However you should not touch the master with alice code, but rather create a alice/416 branch and have ALICE specific commits there. This way we can make sure we always know what is ALICE and what is upstream.